tools/coverage_client.py

```diff
@@ -0,0 +1,1202 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
This file is the `jerry-client-ws.py` from JerryScript. If you have any modifications in the source code, please add js-remote-test copyright into this file.